tls/s2n_psk.c

```diff
@@ -289,3 +289,21 @@ S2N_RESULT s2n_finish_psk_extension(struct s2n_connection *conn)
     GUARD_RESULT(s2n_psk_write_binder_list(conn, &partial_client_hello, client_hello));
     return S2N_RESULT_OK;
 }
+
+S2N_RESULT s2n_connection_set_psks(struct s2n_connection *conn, struct s2n_pre_shared_key *psk_vec, size_t *psk_vec_length)
```

Comment:
We might want to actually call this "s2n_connection_set_external_psks" to make sure nobody tries to use it for resumption secrets.